### ISSUE TITLE
Unable to login whatsoever

### ISSUE THREAD

**Issue by LeilaSchooley** [stanvanrooy/instauto#221]
**Describe the bug**
I'm encountering an issue when using the Instauto library. Specifically, when I reuse the login code from the example provided, it does not log me in, and I receive an empty array when trying to fetch my followers.

 Use the following code snippet to log in:

```
from instauto.api.client import ApiClient
client = ApiClient(username='your_username', password='your_password')
client.log_in()
```

Save the session to a file:
```

client.save_to_disk('your-savefile.instauto')
Attempt to initiate the client from the saved file:
client = ApiClient.initiate_from_file('your-savefile.instauto')
```

```


from instauto.helpers.friendships import get_followers
followers = get_followers(client, username='your_username', limit=100)

```
Expected Behavior:
I expected the code to successfully log me in and retrieve my followers' list.

Setup:

    Instauto version: V2.0.8
    Device profile: Default profile configuration, unchanged
    Instagram profile: It does not have 2FA enabled, but despite this I cant login

Additional Context:
I have tried multiple code snippets to log in and retrieve followers, but none of them have worked as expected. Additionally, I encountered issues with 2FA accounts not working but that would be a separate report

OS: Windows 11

**Comment by LeilaSchooley** [comment]
from instauto.api.client import ApiClient
from instauto.helpers.friendships import get_followers

client = ApiClient(username='', password='')
client.log_in()
client.save_to_disk('your-savefile.instauto')
followers = get_followers(client, username='visitlondon', limit=100)
print(followers)


i try this exact code, the result is an empty array.
no message is shown.

**Comment by stanvanrooy** [comment]
@LeilaSchooley this is the resolved in the latest version, just published to PyPi (V2.1.0). Please try upgrading (`pip install instauto --upgrade`) and try again.
